pkg/cvo/status.go

```diff
@@ -367,6 +368,8 @@ func convertErrorToProgressing(history []configv1.UpdateHistory, now time.Time,
 	switch uErr.UpdateEffect {
 	case payload.UpdateEffectNone:
 		return uErr.Reason, fmt.Sprintf("waiting on %s", uErr.Name), true
+	case payload.UpdateEffectReport:
+		return uErr.Reason, fmt.Sprintf("waiting on %s", uErr.Name), true
```

Comment:
As mentioned [here][1], we don't want to actually wait on these.  You've changed the returned `ok` from `false` to `true`, which is good, but still have `waiting on ...`.  We probably want to just return `uErr.Error()` here for `message` or something.

[1]: https://github.com/openshift/cluster-version-operator/pull/662#discussion_r720535306

Reply:
oh, wait, I think we do want `false` here for `ok` / `skipFailure`.  Because we do want to go `Failing=True` on these (that's the explicit as in [rhbz#1951835][1]).  We just don't want them to block the transition out of the installing state.  And that transition is happening in the `cr.Complete()` call discussed [here][2], and not in this `convertErrorToProgressing` call chain.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1951835
[2]: https://github.com/openshift/cluster-version-operator/pull/662#discussion_r732424001